modules/core/share_limits.py

```diff
@@ -570,7 +570,7 @@ def _has_reached_seeding_time_limit():
         if last_active is not None:
             if not _has_reached_last_active_time_limit():
                 return body
-        if max_ratio is not None:
+        if max_ratio is not None or max_ratio != -1:
```

Comment:
Wouldn't this be an and statement?
For example -1 would eval as true for the "is not None", or am i still half asleep

Reply:
oops you're absolutely right. It's early for me so I'm the one that's half asleep 😅